src/CoreEditor/hello.html

```diff
@@ -0,0 +1,3 @@
+<html>
```

Comment:
We don't need this file

Reply:
It's a test file from my side. By mistake uploaded.